src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -193,12 +196,12 @@ export class HuiStackCardEditor extends LitElement
     fireEvent(this, "config-changed", { config: this._config });
   }
 
-  private _handleMove(ev) {
+  private _handleMove(direction: number) {
     if (!this._config) {
       return;
     }
     const source = this._selectedCard;
-    const target = ev.target.id === "move-before" ? source - 1 : source + 1;
```

Comment:
If you change `ev.target` to `ev.currentTarget` the old code will just work.

Reply:
The old code is not good though :)

Reply:
string comparison vs just adding the 1/-1 to a property is probably better

Reply:
yes, but that still requires `currentTarget` :smile:

Reply:
true

Reply:
ok, so `currentTarget` in function and a `.move` property right?

Reply:
done.
@zsarnett is there a way to show a tooltip for the `mwc-icon-button`? I've added labels, but it isn't showing. The icons are self-explanatory, but a tooltip would be a nice addition.
There is [an issue](https://github.com/material-components/material-components-web-components/issues/1499) in MWC repo, but sadly the status of the tooltip component is TBD.

Reply:
use the `title` attribute

Reply:
done